File: src/main/java/ru/translater/common/Task.java
package ru.translater.common;

import com.github.rvesse.airline.HelpOption;
import com.github.rvesse.airline.annotations.Command;
import com.github.rvesse.airline.annotations.Option;
import lombok.Data;

import javax.inject.Inject;

@Data
@Command(name = "task", description = "Task for execution")
public class Task {
    @Inject
    private HelpOption<Task> help;

    @Option(name = {"-n", "--name"}, title = "Name", arity = 1, description = "An option that takes an argument")
    private String name;
}

File: src/main/kotlin/com/translater/common/model/Task.kt
package com.translater.common.model

import com.github.rvesse.airline.HelpOption
import com.github.rvesse.airline.annotations.Command
import com.github.rvesse.airline.annotations.Option
import javax.inject.Inject

@Command(name = "task", description = "Task for execution")
data class Task(
    @Inject
    var help: HelpOption<Task?>? = null,

    @Option(name = ["-n", "--name"], title = ["Name"], arity = 1, description = "An option that takes an argument")
    var name: String? = null,
)